### PR TITLE
Add new query parameters for containerized environments to the api

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to #1625 

## This introduces a breaking change

- [x] No

## This Pull Request implements

Add new query parameters for container images to the OpenAPI specification

### CODE REVIEW

**Comment by mayaCostantini** [comment]
/hold 
Needs generating changes with swagger-codegen

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -558,6 +558,10 @@ paths:
       - $ref: "#/components/parameters/python_version"
       - $ref: "#/components/parameters/cuda_version"
       - $ref: "#/components/parameters/image_name"
+      - $ref: "#/components/parameters/library_name"
+      - $ref: "#/components/parameters/symbol"
+      - $ref: "#/components/parameters/package_name"
+      - $ref: "#/components/parameters/rpm_package_name"
```
Comment (fridex) [inline openapi/openapi.yaml:564]: These references require definition in `.components.parameters` section.

Comment (mayaCostantini) [reply openapi/openapi.yaml:564]: Added now, thank you! :100:

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**